src/helpers/timers.ts

```diff
@@ -2,6 +2,8 @@
 /* globals jest */
 const globalObj = typeof window === 'undefined' ? global : window;
 
+// trigger pipeline
+
```

Comment:
looks like it worked this time. I wholeheartedly recommend using `git commit --allow-empty` for such cases when playing with CI (we all love it, don't we?) :D 
```suggestion
```

Reply:
Aha didn't think about that, was planning to remove the commit and push force^^